fastlane/docs/FAQs.md

```diff
@@ -90,6 +90,9 @@ lane :deploy_all do
 end
 ```
 
+And you can combine multiple envs in one go
+Ex: `fastlane build --env app1,env1,env2` will use `.env.app1` `.env.env1` and `.env.env2`
```

Comment:
Those file look awfully similar. Are they meant to be the same? Why not sync them automatically?